server/services/reportedAdjudicationsService.ts

```diff
@@ -479,7 +481,7 @@ export default class ReportedAdjudicationsService {
       },
       {
         label: 'Location',
-        value: `${location?.userDescription || ''}`,
+        value: `${location?.userDescription || ''}, ${agencyName.description}`,
```

Comment:
Do we have a test which covers `location` being null and so the `value` only equals `agencyName.description`?

Reply:
Embarrassingly, now that I check - there's no unit tests at all for this function 😱 I'll add some now - but yeah you're right, if location was null it's going to look a little odd with that comma and the agency